multi-agent/run/fixtures/seeder.py

```
@@ -0,0 +1,83 @@
+        logger.info("No fixtures directory found at %s — skipping seed.", FIXTURES_DIR)
+        return 0
+
+    yaml_files = sorted(FIXTURES_DIR.glob("*.yml")) + sorted(FIXTURES_DIR.glob("*.yaml"))
+    if not yaml_files:
+        logger.info("No YAML fixtures found in %s", FIXTURES_DIR)
+        return 0
+
+    inserted = 0
+    for path in yaml_files:
+        try:
+            with open(path, "r") as f:
```

Ruff (0.15.15)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)